.github/workflows/release.yml

```diff
@@ -13,13 +13,6 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
         with:
-          # Fetch git history with a depth of 50 commits so that Changesets can generate
-          # changelogs with the correct commits information.
-          # Note: this means that if we have merged >50 PRs since the last release,
-          # the changelogs might not contain all correct information.
-          # We could set `fetch-depth: 0`, but that would cause the entire history
-          # to be cloned. Using 50 seems like a good balance to start with.
-          fetch-depth: 50
```

Comment:
See https://github.com/atlassian/changesets/releases/tag/%40changesets%2Fcli%402.13.0

Reply:
Thanks for sharing!